components/scream/tests/uncoupled/shoc/shoc_standalone.cpp

```diff
@@ -23,7 +23,8 @@ TEST_CASE("shoc-stand-alone", "") {
   // Load ad parameter list
   std::string fname = "input.yaml";
   ekat::ParameterList ad_params("Atmosphere Driver");
-  REQUIRE_NOTHROW ( parse_yaml_file(fname,ad_params) );
+  //REQUIRE_NOTHROW ( parse_yaml_file(fname,ad_params) );
```

Comment:
This might be better handled like this:
```c++
#ifndef KOKKOS_ENABLE_HIP
REQUIRE_NOTHROW(
#endif
parse_yaml_file(fname,ad_params)
#ifndef KOKKOS_ENABLE_HIP
)
#endif
;
```

Reply:
I think we can remove `REQUIRE_NOTHROW` altogether. It's a silly test, since a throw will already make the test fail...

Reply:
thanks -- though i replaced it with 
  #ifndef KOKKOS_ENABLE_HIP
  REQUIRE_NOTHROW ( parse_yaml_file(fname,ad_params) );
  #else
  parse_yaml_file(fname,ad_params);
  #endif